haystack/utils/openai_utils.py

```diff
@@ -112,7 +112,7 @@ def openai_request(
     url: str,
     headers: Dict,
     payload: Dict,
-    timeout: Union[float, Tuple[float, float]] = OPENAI_TIMEOUT,
+    timeout: Union[float, Tuple[float, float]] = None,
```

Comment:
Now that the default value is `None`, the type of `timeout` should be made `Optional`, this should also fix the tests

Reply:
done